go.mod

```diff
@@ -639,6 +639,8 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/journ
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/zookeeperreceiver => ./receiver/zookeeperreceiver
 
+replace github.com/open-telemetry/opentelemetry-collector-contrib/receiver/apachereceiver => ./receiver/apachereceiver
```

Comment:
This is unrelated to couchdb likely needs to be removed.

Reply:
will remove